src/ai/ai_gui.cpp

```diff
@@ -190,7 +190,7 @@ struct AIListWindow : public Window {
 	{
 		switch (widget) {
 			case WID_AIL_LIST: { // Select one of the AIs
-				int sel = this->vscroll->GetScrolledRowFromWidget(pt.y, this, WID_AIL_LIST, 0, this->line_height) - 1;
+				int sel = this->vscroll->GetScrolledRowFromWidget(pt.y, this, WID_AIL_LIST) - 1;
```

Comment:
I attempted double clicking and it resulted in a division by zero.
The passed line_height was -1, but got set to 0 because wid->resize_y is 0.

>	openttd.exe!Window::GetRowFromWidget(int clickpos=85, int widget=5, int padding=0, int line_height=0) Line 206	C++
 	openttd.exe!Scrollbar::GetScrolledRowFromWidget(int clickpos=85, const Window * const w=0x000001ec5d82a460, int widget=5, int padding=0) Line 1971	C++
 	openttd.exe!AIConfigWindow::OnClick(Point pt={...}, int widget=5, int click_count=1) Line 898	C++